contracts/protocol/libs/ProtocolLib.sol

```diff
@@ -174,6 +174,10 @@ library ProtocolLib {
         mapping(uint256 => BosonTypes.Condition) exchangeCondition;
         // groupId => offerId => index on Group.offerIds array
         mapping(uint256 => mapping(uint256 => uint256)) offerIdIndexByGroup;
+        // seller id => Seller
+        mapping(uint256 => BosonTypes.Seller) sellerPendingUpdates;
+        // seller id => AuthToken
+        mapping(uint256 => BosonTypes.AuthToken) sellerPendingAuthTokenUpdate;
```

Comment:
I'd like to suggest alternatives for these mappings that are more akin to the "X by Y" naming convention.

- `pendingAddressUpdatesBySeller`
- `pendingAuthTokenUpdatesBySeller`

Reply:
renamed